src/main/java/in/projecteka/consentmanager/clients/PatientServiceClient.java

```diff
@@ -1,13 +1,16 @@
 package in.projecteka.consentmanager.clients;
 
 import in.projecteka.consentmanager.clients.model.LinkedCareContexts;
+import in.projecteka.consentmanager.link.link.model.PatientLinksResponse;
 import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.reactive.function.client.WebClient;
 import reactor.core.publisher.Mono;
 
 import java.util.function.Supplier;
 
+import static com.google.common.net.HttpHeaders.AUTHORIZATION;
```

Comment:
can you use import static org.springframework.http.HttpHeaders.AUTHORIZATION; instead of google import

Reply:
Sure, will do that.

Reply:
That was just an unused import, removed that.